src/GlobalStyles.js

```diff
@@ -1,6 +1,16 @@
 import { createGlobalStyle } from 'styled-components'
 
 export default createGlobalStyle`
+html{
```

Comment:
Really cool that you use CSS variables. One thing though. According to MDN it is best practice to store them in a pseudo-element `:root` as it is more specific than `html`. 
```suggestion
:root {
```
You can read more about it here :)
https://developer.mozilla.org/en-US/docs/Web/CSS/:root
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
Yeah, i was unsure, because i saw an example where someone explained that he sets the variables to html in react/styled-componets. but i will change that :D